modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -202,9 +207,83 @@ const DocumentLibrary = ({
 	);
 };
 
+const Upload = ({
```

Comment:
@alinedoleron, how about name it to `GuestUploadFile` to make more explicit the purpose of this component?